contrib/epee/include/net/abstract_tcp_server2.inl

```diff
@@ -482,7 +476,7 @@ PRAGMA_WARNING_DISABLE_VS(4355)
     if (m_ssl_support == epee::net_utils::ssl_support_t::e_ssl_support_enabled)
     {
       // Handshake
-      if (!handshake(boost::asio::ssl::stream_base::server))
```

Comment:
`bytes_transferred` -> `buffer_ssl_init_fill` in case it took multiple calls to read the beginning portion of the handshake.